lib/board.ex

```diff
@@ -72,19 +77,19 @@ defmodule Board do
     |> Enum.any?(&is_winning_vector?/1)
   end
 
-  @spec get_both_diagonals_status(map()) :: boolean()
-  defp get_both_diagonals_status(board) do
-    board_dimensions = get_board_dimensions(board)
+  @spec both_diagonals_status(map()) :: boolean()
```

Comment:
Just throwing out ideas for an alternative name, maybe `win_by_diagonals?`?

Reply:
Same as above, I think `win_by_diagonals` is a much better name!